man/tte_source_objects.Rd

```diff
@@ -15,7 +15,31 @@
 \alias{ae_sev_event}
 \alias{ae_wd_event}
 \title{Pre-Defined Time-to-Event Source Objects}
-\format{An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.}
+\format{
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{censor_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
+
+An object of class \code{event_source} (inherits from \code{tte_source}, \code{source}, \code{list}) of length 5.
```

Comment:
@bundfussr @bms63 this looks a bit weird?? i didn't even touch the program for this. not sure if something went wrong with my devtools run there. want me to revert this one back to the previous version?